lib/cloud_controller/config.rb

```diff
@@ -42,6 +34,22 @@ def schema_class_for_context(context, _config)
         const_get("VCAP::CloudController::ConfigSchemas::#{module_name}::#{context.to_s.camelize}Schema")
       end
 
+      def read_file(file_name)
+        deep_symbolize_keys_except_in_arrays(VCAP::CloudController::YAMLConfig.safe_load_file(file_name)) || {}
+      end
+
+      def load_from_hash(config_hash, context: :api, secrets_hash: {})
+        config = deep_symbolize_keys_except_in_arrays(config_hash)
```

Comment:
You can remove this `deep_symbolize_keys_except_in_arrays` line from here as it is already done in `read_file`.

Reply:
We need to keep this, first it is needed for deep merge with secret_hash as it expect a valid hash `config = config.deep_merge(secrets_hash)`, second, `deep_symbolize_keys_except_in_arrays` returns a hash if even passed an already hash.